src/com/pentalog/pentastagiu/homework2/Board.java

```diff
@@ -0,0 +1,86 @@
+package com.pentalog.pentastagiu.homework2;
+
+import java.util.*;
+
+
+public class Board {
```

Comment:
este ok, dar este un scenariu simplu cel pe care l-ai implementat, ti-as sugera sa folosesti un while pentru citirea de la tastatura si sa creezi un scenariu mai complex; in care sa ai optiunile: 1.Register user, 2.Post message(caz in care o sa ceri detalii legate de ce user, ce mesaj baga), 3.Display all messages si astfel sa poti sa faci register la 2 useri, sa inregistrezi cate 2 mesaje pentru fiecare si la sfarsit cand afisezi mesajele sa iti apara toate cele 4